modules/dnn/cmake/download_google_inception5.cmake

```diff
@@ -0,0 +1,30 @@
+function(download_google_inception5 dst_dir status_var)
```

Comment:
`OPENCV_TEST_DATA_PATH` is specified manually before OpenCV test launch.

1.
> this variable may be not defined - do I have to skip test in this case?

OpenCV C++ tests fail in this case. They are "skipped" in case of external data only.
Unfortunately we don't have Java tests that uses `OPENCV_TEST_DATA_PATH` (in C++ tests test data is managed by "opencv_ts" module - via `cvtest::findDataFile()` call).

2.
`OPENCV_DNN_TEST_DATA_PATH` points to additional optional data (large files with DNN models).
They can be downloaded by this [script](https://github.com/opencv/opencv_extra/blob/master/testdata/dnn/download_models.py) into separate directory ("OPENCV_DNN_TEST_DATA_PATH/dnn", size is 2.1Gb).